### ISSUE TITLE
HAND evaluation strategy

### ISSUE THREAD

**Issue by echeipesh** [azavea/noaa-flood-mapping#48]
It is this projects goal to evaluate if HAND is a valuable feature to add for detecting flood inundation. 
The original plan was to [combine HAND with Sen1Floods11](https://github.com/azavea/noaa-flood-mapping/issues/5) and see if the model performance is improved.

However, on slightly further thought we realized that HAND intersects only the USA subset of Sen1Floods11 which likely will not be sufficient training data to actually answer the question posed above. We need a new strategy of how we might construct an experiment that reveals this information.

**Comment by jamesmcclain** [comment]
I assume that the goal is to determine whether HAND + Sentinel-1 does better than just Sentinel-1 in Sen1Floods11 type experiments/workloads.

Since we identified that the intersection of HAND and Sen1Floods11 is (probably) too small to permit a conclusion, I guess two possible first steps would be to increase the area of HAND (presumably by switching or substituting) and/or increase the area of Sen1Floods11.

Increasing the area ofSen1Floods11-like data might include using the [GLOFIMR](https://sdml.ua.edu/glofimr/) dataset suggested by @moradology .  Alternatively, a DEM + Sentinel-1 on a large-enough area could give enough of a signal to give us confidence that HAND will provide a noticeable benefit.

It seems like checking whether the intersection of HAND and GLOFIMR meets our needs is something to determine.

**Comment by echeipesh** [comment]
This is my take on our conversation:

## Objective 
Our main objective is to ascertain if HAND is a useful feature in a model to detect flood inundation on S1 scenes. The most direct goal is to have an experimental result of model performance on the same dataset with HAND and without HAND.

## Alternative
USFIMR intersects heavily with CONUS and HAND. We can train and compare two models:
1. S1 features with USFIMR labels
2. S1, HAND features with USFIMR labels

This gives us pretty good comparison.

## Sen1Floods11 comparison

We might still be tempted to relate the improvement to Sen1Floods11.
This is possible because one flood from Sen1Floods11 is in CONUS/Kentucky, we can use that as hold-out set for validation.
Here we can train 3 models:

1. Sen1Floods11 model using Kentucky as validation set
  * we'd probably want to train using weak labels and hand validated labels, so this is really 2 models
2. USFIMR, S1
3. USFIMR, S1, HAND

For all three we can use the S1 chips from Sen1Floods11 as validation.

HOWEVER, before doing this we better review that our S1 chips (which we're planning to source from Sentinel Hub) are actually compatible with S1 chips used with Sen1Floods11 generated by GEE.

**Comment by jamesmcclain** [comment]
> USFIMR intersects heavily with CONUS and HAND. We can train and compare two models:
> 
> 1. S1 features with USFIMR labels
> 2. S1, HAND features with USFIMR labels

Yes, I am in full agreement with this.  I would like to add a third "S1, DEM features with USFIMR labels" just to measure the extent to which HAND > DEM.

> ## Sen1Floods11 comparison
> We might still be tempted to relate the improvement to Sen1Floods11.
> This is possible because one flood from Sen1Floods11 is in CONUS/Kentucky, we can use that as hold-out set for validation.
> Here we can train 3 models:
> 
> 1. Sen1Floods11 model using Kentucky as validation set
> 
> * we'd probably want to train using weak labels and hand validated labels, so this is really 2 models
> 
> 1. USFIMR, S1
> 2. USFIMR, S1, HAND

I think this gets the utility of USFIMR compared to sen1floods11?  Yes, I think that is important.  I just for completeness, I would like to add USFIMR, S1, DEM

> HOWEVER, before doing this we better review that our S1 chips (which we're planning to source from Sentinel Hub) are actually compatible with S1 chips used with Sen1Floods11 generated by GEE.

Very important, also.  If you can provide chips for the sen1floods11 hand-labeled `test` set and `bolivia` set, it will be possible to do a direct comparison (run existing models on the alternative Sentinel-1 imagery to ensure that the results are no worse).

**Comment by jamesmcclain** [comment]
sen1floods11 + DEM results can be found [here](https://github.com/azavea/noaa-flood-mapping/issues/51).
